skeleton/local-deploy/deployment/model_servers/object_detection_python/README.md

```diff
@@ -0,0 +1,52 @@
+# Object_Detection_Python Model Server
```

Comment:
Is there a way we can exclude the other model servers folders that weren't selected by the user? Perhaps via nunjucks if statements?